modules/end-user-guide/nav.adoc

```diff
@@ -1,49 +1,49 @@
 .End-user Guide
 
-* xref:navigating-che.adoc[]
-** xref:navigating-che-using-the-dashboard.adoc[]
-** xref:importing-certificates-to-browsers.adoc[]
-** xref:accessing-che-from-openshift-developer-perspective.adoc[]
-* xref:che-theia-ide-basics.adoc[]
-** xref:defining-custom-commands-for-che-theia.adoc[]
-** xref:version-control.adoc[]
-** xref:che-theia-troubleshooting.adoc[]
-** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[]
+// * xref:navigating-che.adoc[]
+// ** xref:navigating-che-using-the-dashboard.adoc[] // NOT NECESSARY
+// ** xref:importing-certificates-to-browsers.adoc[] // OBSOLETE
+* xref:accessing-che-from-openshift-developer-perspective.adoc[]
+// * xref:che-theia-ide-basics.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:defining-custom-commands-for-che-theia.adoc[] // REDIRECT TO https://devfile.io/docs/devfile/2.0.0/user-guide/adding-commands-to-a-devfile/
+// ** xref:version-control.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:che-theia-troubleshooting.adoc[] // REMOVE
+// ** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[] // REMOVE
 * xref:workspaces-overview.adoc[]
-** xref:creating-a-workspace-from-a-code-sample.adoc[]
+// ** xref:creating-a-workspace-from-a-code-sample.adoc[] // REMOVE USELESS
 ** xref:creating-a-workspace-from-a-template-devfile.adoc[]
```

Comment:
I don't think this is still a viable option https://github.com/eclipse-che/che-docs/blob/master/modules/end-user-guide/partials/assembly_creating-a-workspace-from-a-template-devfile.adoc

AFAICT, there are two options: load from git, or open a sample:

![image](https://user-images.githubusercontent.com/227597/158839467-3312d05b-39ef-4810-96b0-cdbd6ddff58c.png)